src/tilda.c

```diff
@@ -766,7 +806,8 @@ int main (int argc, char *argv[])
 #endif
     /* Parse the command line */
     config_file = NULL;
-    need_wizard = parse_cli (argc, argv, &config_file);
+    struct tilda_cli_opts *cli_opts = init_cli_opts_struct();
```

Comment:
I do not like `opts`, please use `options` everywhere and drop the `_struct` suffix from the function name.